Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -0,0 +1,155 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class PledgeDescriptionCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var containerImageView: UIView = { UIView.init() }()
+  private lazy var pledgeImage: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var descriptionStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var estimatedDeliveryLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var dateLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var descriptionLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var learnMoreLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var spacerView: UIView = { UIView(frame: .zero) }()
+
+  func configureWith(value: String) {
+    _ = self.dateLabel
+      |> \.text .~ value
+  }
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    _ = (self.rootStackView, self.contentView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    self.containerImageView.widthAnchor.constraint(equalToConstant: 100.0).isActive = true
+    self.rootStackView.addArrangedSubview(self.containerImageView)
+
+    self.pledgeImage.widthAnchor.constraint(equalToConstant: 90.0).isActive = true
+    self.pledgeImage.heightAnchor.constraint(equalToConstant: 130.0).isActive = true
+    self.containerImageView.addSubview(self.pledgeImage)
+    self.pledgeImage.centerXAnchor.constraint(equalTo: self.containerImageView.centerXAnchor).isActive = true
+
+    self.arrangeStackView()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> \.backgroundColor .~ UIColor.hex(0xf0f0f0)
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+
+    _ = self.containerImageView
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+      |> \.backgroundColor .~ UIColor.blue
+
+    _ = self.spacerView
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+
+    _ = self.pledgeImage
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+      |> \.backgroundColor .~ UIColor.orange
+
+    _ = self.descriptionStackView
+      |> descriptionStackViewStyle
+
+    _ = self.estimatedDeliveryLabel
+      |> estimatedDeliveryLabelStyle
+
+    _ = self.dateLabel
+      |> dateLabelStyle
+
+    _ = self.descriptionLabel
+      |> descriptionLabelStyle
+
+    _ = self.learnMoreLabel
+      |> learnMoreLabelStyle
+  }
+
+  private func arrangeStackView() {
+    self.spacerView.heightAnchor.constraint(equalToConstant: 10.0).isActive = true
+    self.descriptionStackView.addArrangedSubview(self.spacerView)
```

Comment:
Again, whatever you feel is more readable, but this can be also written as:

```swift
[
  self.spacerView,
  self.estimatedDeliveryLabel,
  self.dateLabel,
  self.descriptionLabel,
  self.learnMoreLabel
]
.forEach(self.descriptionStackView.addArrangedSubview)
```

Perhaps makes grouping the views to be added a bit nicer, I wouldn't really do it for just one view though.